includes/Frontend/Payment.php

```
@@ -492,24 +496,24 @@ public function send_to_gateway() {
      */
     public static function insert_payment( $data, $transaction_id = 0, $recurring = false ) {
         global $wpdb;
-        
+
         $user_id = get_current_user_id();
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$user_id'. (undefined)

(UnusedLocalVariable)